modules/standard/IO.chpl

```diff
@@ -3632,6 +3632,20 @@ private proc warnBinaryRead(type t, param depth : int) throws {
 
   See :ref:`the serializers technote<ioSerializers>` for a general overview
   of Serializers and their usage.
+
+  .. warning::
+
+    In the 1.32 release this format included bytes representing the length of
```

Comment:
I can't remember, does this apply to 1.31 as well? If so, maybe "prior to the 1.33 release"?

Reply:
Technically these types weren't stable or in the IO module until 1.32.